proto/src/attribute.rs

```diff
@@ -22,6 +22,8 @@ pub enum Attribute {
     AcpCreateClass,
     AcpEnable,
     AcpModifyClass,
+    AcpModifyPresentClass,
+    AcpModifyRemoveClass,
```

Comment:
It's "removed" on attr two lines below, can we be consistent please 🤔

Reply:
It shouldn't have been removed in the first place, but we also can't change it now. I think this makes more sense.